### PR TITLE
Add NPM scripts

### PR DESCRIPTION
**Pull request by chris-kobrzak**
Issue #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 ```
-npm install webpack -g
-npm install karma-cli -g
 npm install
 ```
 
 ## Workflow
-1. Run webpack-dev-server.js with Node. This will run `webpack-dev-server` with
-HMR enabled on port 5000:
+1. Run `webpack-dev-server` acting as a static content HTTP server with hot
```
Comment (katranci) [inline README.md]: It would be more correct to say "acting as asset server".

File: package.json [hunk 2]
```diff
@@ -27,8 +28,12 @@
     "webpack-dev-server": "^1.10.1"
   },
   "scripts": {
```
Comment (katranci) [inline package.json]: I think some of these scripts don't match with the README.

**Comment by katranci** [comment]
Looks good :+1:

**Comment by chris-kobrzak** [comment]
Resolves #30

File: package.json [hunk 3]
```diff
@@ -27,8 +28,12 @@
...
+    "browse": "open http://localhost:5001",
     "build": "webpack",
     "clean": "rm -fr tmp/coverage/javascript/* && rm -fr dist/*",
+    "dev-server": "node webpack-dev-server.js",
+    "start": "http-server -p 5001",
+    "lint": "eslint --ignore-pattern dist ./; exit 0",
```
Comment (chris-kobrzak) [inline package.json]: More background on the `exit 0` hack: https://github.com/eslint/eslint/issues/2409#issuecomment-103768546

**Pull request merged**